lesson-3.js

```diff
@@ -0,0 +1,38 @@
+// Task 1
+let n = 100; i = 1;
+const arr = [];
+while (i <= n) {
+  let j = 2;
+  let isSimple = true;
+  while (isSimple && j < i) {
+    if (i % j === 0) isSimple = false;
+      j++;
+  }
+  if (isSimple) arr.push(i);
+  i++;
+}
+
+//Task 2 and 3
+let cart =[
+  ['Куртка',1,5000],
```

Comment:
Не забывай о пробелах между операторами и операндами
```suggestion
  ['Куртка', 1, 5000],
```